backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/freinet/FreinetDataImporter.kt

```diff
@@ -0,0 +1,13 @@
+package xyz.elitese.ehrenamtskarte.importer.freinet
+
+import xyz.elitese.ehrenamtskarte.importer.general.HttpDownloadHelper
+
+object FreinetDataImporter {
+
+    fun import() {
+        val url = FreinetDataImporter::class.java.getResource("/freinet_import/secrets/import_all.txt").readText()
```

Comment:
I think it would be nice to have this configurable via the system properties.

Reply:
You mean the url in general?